draft-wang-ppm-dap-taskprov.md

```diff
@@ -237,7 +237,7 @@ task_id = HKDF-Extract(task_prov_salt, task_config)
 
 where `task_config` is the `TaskConfig` structure disseminated by the Author.
 String `task_prov_salt` is defined to be the SHA-256 hash of the octet string
-"dap-taskprov-00". Function HKDF-Extract() denotes the HKDF extraction function
+"dap-taskprov". Function HKDF-Extract() denotes the HKDF extraction function
```

Comment:
I have no objection to this change. If we need a hard-break between versions, then we can always update the extension codepoint.